pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -127,3 +127,234 @@ type LeaderElectionConfiguration struct {
 	// leader election is enabled.
 	RetryPeriod unversioned.Duration `json:"retryPeriod"`
 }
+
+type KubeletConfiguration struct {
+	unversioned.TypeMeta
+
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
Is this relative or absolute? If relative, is it resolved relative to the location of the file containing this config, or to the cwd of the kubelet process, or something else?